style.css

```diff
@@ -431,6 +375,13 @@ nav #mylogo {
   top: 1%;
 }
 
+.drop-box {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  right: 4%;
+}
+
```

Comment:
- [ ] Please notice that the technologies or languages cards in the About me sections are placed vertically, they are supposed to be aligned horizontally please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/221256708-1dc9507d-bd1a-444c-8424-00d4d6f46066.png)

how they should be :point_down: 
![image](https://user-images.githubusercontent.com/97521557/221256789-5d0fe186-1e5e-48de-8aae-dc8ae370fc90.png)